scripts/pre_redeploy.sh

```diff
@@ -142,3 +142,7 @@ if [[ ${RPC_TARGET_CHECKOUT} == "r14.7.0" ]]; then
      patch -p1 < ${RPC_UPGRADES_DEFAULT_FOLDER}/playbooks/patches/newton/rpc-o-14.7.0-ansible-role-requirements.patch
    popd
 fi
+
+#RLM-268 Manually give true value to the logic so cinder CLI can have --insecure flag
+mkdir -p /etc/openstack_deploy/group_vars
+echo 'keystone_service_adminuri_insecure: true' | tee -a /etc/openstack_deploy/group_vars/cinder_all.yml
```

Comment:
Should this be idempotent?  As written, this will (try to) create `group_vars` dir and append to `cinder_all.yml` each time it's invoked.

Reply:
Yeah, you're right. 'mkdir -p' can keep idempotent but 'echo' needs a pre check for the key-value pair.